src/jit/emitxarch.cpp

```diff
@@ -11996,6 +12027,13 @@ size_t emitter::emitOutputInstr(insGroup* ig, instrDesc* id, BYTE** dp)
                 code = insEncodeRMreg(ins, code);
                 mReg = id->idReg2();
                 rReg = id->idReg1();
+#ifndef LEGACY_BACKEND
+                if (ins == INS_extractps)
+                {
+                    mReg = id->idReg1();
```

Comment:
Why can't we handle this in codegen?

Reply:
Hmm..we can. The special case `extractps` uses `ModRM:r/m` for the `dst` register (store semantics). 
I will handle this in codgen, thanks!